lib/experimental/new_embed.dart

```diff
@@ -804,12 +878,129 @@ class AnalysisResultsController {
   }
 }
 
+class ConsoleController {
```

Comment:
A one or two line class comment explaining would be probably be a good idea.

We could use that on the rest of the classes, too, of course, but that's a whole other ball of wax.

Reply:
Good call, done